shared/logger/types.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+// Separate type file because standard is failing
```

Comment:
not sure what this comment means

Reply:
hmm me neither. I remember trying to do something with opaque types and eslint failing at that. This probably doesn't mean anything.

Reply:
O it means you can't move these types into another file because the interface definition messes up with standard formatting I think ?